src/libdrakvuf/vmi.c

```diff
@@ -1748,6 +1759,7 @@ bool init_vmi(drakvuf_t drakvuf, bool fast_singlestep)
     drakvuf->memaccess_lookup_trap = g_hash_table_new_full(g_direct_hash, g_direct_equal, NULL, NULL);
     drakvuf->remapped_gfns = g_hash_table_new_full(g_direct_hash, g_direct_equal, NULL, free_remapped_gfn);
     drakvuf->remove_traps = g_hash_table_new_full(g_direct_hash, g_direct_equal, NULL, NULL);
+    drakvuf->injections_in_progress = g_hash_table_new_full(g_direct_hash, g_direct_equal, NULL, NULL);
```

Comment:
This could be just a g_hash_table_new, not the full version (same for some of the ones above I guess).

Reply:
Fixed